README.md

```diff
@@ -31,6 +31,8 @@ Sway is an incredible window manager, and certainly one of the most well establi
     - `blur_xray enable|disable`
     - `blur_passes <integer value 0 - 10>`
     - `blur_radius <integer value 0 - 10>`
++ LayerShell effects: *ONLY ON SWAYFX-GIT, NOT YET RELEASED*
+    - `layer_effects <layer namespace, eg: "waybar"> blur shadow corner_radius`
```

Comment:
not sure how, but we should indicate that a user can include any of the options between `blur`, `shadow`, and `corner_radius`

Reply:
Would something like this be better?:

```markdown
+ LayerShell effects: *ONLY ON SWAYFX-GIT, NOT YET RELEASED*
    - `layer_effects <layer namespace, eg: "waybar"> <effects, eg: blur shadow corner_radius>`
```